### DISCUSSION TITLE
can i put `lazy.lock` to `.gitignore`?

### DISCUSSION THREAD

**Discussion by cofeek-codes** [LazyVim/LazyVim#4403] [Q&A] [answered]
Half of  commits in my config repo is related to just regulary updating dependencies, so i started wondering if i am doing things the right way. Is it good solution to put `lazy.lock` file to `.gitignore`

**Comment by zodman** [comment]
if you dont track your `lazy.lock` , its like don't track the `package-lock.json` track it ...

**Reply by cofeek-codes** [reply]
@zodman ok, but is there any other way to avoid "update commits"?

**Reply by dpetka2001** [reply, marked as answer]
You either want to track it or not. If you track it and the file changes then it will show in the `git status`. You can just commit the other files and not this one. And just commit this one file once per week. I really don't understand what the problem you're trying to solve is. I also track it, but only commit that file from the working tree once per week.
